students/Daniel_Rodriguez/Lesson02/Assignment/charges_calc.py

```diff
@@ -0,0 +1,161 @@
+"""
+Returns total price paid for individual rentals
+"""
+import argparse
+from argparse import RawTextHelpFormatter
+
+import json
+import datetime
+import math
+import logging
+
+# logging.basicConfig(level=logging.DEBUG)
+
+
+def parse_cmd_arguments():
+    """
+    Parse command line arguments
+    :return:
+    """
+    parser = argparse.ArgumentParser(description='Process some integers.',
+                                     formatter_class=RawTextHelpFormatter)
+
+    parser.add_argument('-i', '--input', help='Input JSON file', required=True)
+    parser.add_argument('-o', '--output', help='Output JSON file',
+                        required=True)
+
+    parser.add_argument('-d', '--debug', help='Debug message level:\n'
+                                              '0: No debug messages or '
+                                              'log file\n'
+                                              '1: Only error messages\n'
+                                              '2: Error messages and warnings\n'
+                                              '3: Error messages, warnings,'
+                                              'and debug messages',
+                        required=False)
+
+    return parser.parse_args()
+
+
+def load_rentals_file(filename):
+    """
+    load_rentals_file
+    :param filename:
+    :return:
+    """
+    logging.debug('Reading input JSON file')
+    logging.debug('Inside load_rentals_file function')
+    logging.debug('Opening Filename: {}'.format(filename))
+
+    with open(filename) as file:
+        try:
+            data = json.load(file)
+            logging.debug('Reading data file...')
+        except Exception as error:
+            logging.error('Source JSON file {} could not be loaded! '
```

Comment:
Is this informative enough? Is it true that we cannot load, or the format might be wrong...